keps/sig-instrumentation/1668-log-tracking/README.md

```diff
@@ -268,232 +130,108 @@ OpenTelemetry has SpanContext which is used for propagation of K8s component.
-expectations). Please adhere to the [Kubernetes testing guidelines][testing-guidelines]
-when drafting this test plan.
+- For request from kubectl, SpanContext is null,  do [Start](https://github.com/open-telemetry/opentelemetry-go/blob/3a9f5fe15f50a35ad8da5c5396a9ed3bbb82360c/sdk/trace/tracer.go#L38)() to start new SpanContext (new trace-id and span-id)
+- For request from controller we can get a valid SpanContext(including trace-id and span-id), do [Start](https://github.com/open-telemetry/opentelemetry-go/blob/3a9f5fe15f50a35ad8da5c5396a9ed3bbb82360c/sdk/trace/tracer.go#L38)() to update the SpanContext (new span-id)
 
-[testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
--->
-TBD
+1.2 Chain SpanContext and initial-trace-id to "r.ctx"
 
-### Graduation Criteria
+- we use r.ctx to propagate those IDs to next handler
 
-<!--
-**Note:** *Not required until targeted at a release.*
+**2. inittrace handler**
 
-Define graduation milestones.
+Implement a new initrace handler to propagate the initial-trace-id.
 
-These may be defined in terms of API maturity, or as something else. The KEP
-should keep this high-level with a focus on what signals will be looked at to
-determine graduation.
+2.1 do our new Extract() to get initial-trace-id from request header
 
-Consider the following in developing the graduation criteria for this enhancement:
-- [Maturity levels (`alpha`, `beta`, `stable`)][maturity-levels]
-- [Deprecation policy][deprecation-policy]
+- For request from kubectl we can't get initial-trace-id,  chain a empty initial-trace-id to a golang ctx
+- For request from controller we can get initial-trace-id, chain the initial-trace-id to a golang ctx
 
-Clearly define what graduation means by either linking to the [API doc
-definition](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning)
-or by redefining what graduation means.
+2.2 chain above golang cxt to r.ctx(updated in above 1.2)
+-  we use r.ctx to propagate those IDs to next handler
 
-In general we try to use the same stages (alpha, beta, GA), regardless of how the
-functionality is accessed.
+**3. Inject SpanContext and initial-trace to request header sent to webhook**
 
-[maturity-levels]: https://git.k8s.io/community/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions
-[deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
+- call othttp's original [Inject](https://pkg.go.dev/go.opentelemetry.io/otel/api/propagators#TraceContext.Inject)() to inject the SpanContext from r.ctx to header
+- call our new Inject() to inject the initial-trace-id from r.ctx to header
 
-Below are some examples to consider, in addition to the aforementioned [maturity levels][maturity-levels].
+### Design of ID propagation (controller)
+**Extract SpanContext and initial-trace-id from annotation of object to golang ctx**
```

Comment:
```suggestion
**1. Extract SpanContext and initial-trace-id from annotation of object to golang ctx**
```
In the above section, the bold sentences have been numbered, but there are not in this section.
I wonder if you forget to number.